### PR TITLE
use id area key

### PR DESCRIPTION
**Pull request by amishas157**
Using id-area-key to create polygons for closed Linestring type features.

cc @ajithranka

### CODE REVIEW

File: parsers/element.js [hunk 1]
```diff
@@ -78,4 +84,12 @@ function ElementParser(json) {
     ).map(createFeature);
 }
 
+function isClosedWay(nodes) {
+  var firstNode = nodes[0];
+  var lastNode = nodes[nodes.length - 1];
+  if (firstNode.lat === lastNode.lat && firstNode.lon === lastNode.lon)
```
Comment (ajithranka) [inline parsers/element.js]: For any pattern like

```js
if (condition) {
  return true;
} else {
  return false;
}
```

you can simplify it as just

```js
  return (condition);
```

File: parsers/element.js [hunk 2]
```diff
@@ -26,7 +27,12 @@ function ElementParser(json) {
       return [node.lon, node.lat].map(parseFloat);
     });
     var properties = R.omit(['nodes'], data);
-    return turf.lineString(geometry, properties);
+
+    if (data.tags && ak.isArea(data.tags) && isClosedWay(data.nodes)) {
+      return turf.polygon([geometry], properties);
+    }
+    else
```
Comment (ajithranka) [inline parsers/element.js]: Just stylistic here, but we should prefer using `{}` around `if/else` blocks.

**Comment by ajithranka** [comment]
This looks great @amishas157! Left some suggestions for small improvements to the code ^.

Ref #1.

**Comment by amishas157** [comment]
Thanks @ajithranka for the review. Making the changes.

**Comment by ajithranka** [comment]
Merged.

**Pull request merged**